mongodbatlas/resource_mongodbatlas_project_api_key.go

```diff
@@ -24,8 +24,9 @@ func resourceMongoDBAtlasProjectAPIKey() *schema.Resource {
 		},
 		Schema: map[string]*schema.Schema{
 			"project_id": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: fmt.Sprintf(DeprecationByVersionMessageParameter, "1.16.0"),
```

Comment:
[if is not a breaking change] Can we add a migration test scenario to make sure that upgrading to the latest version with the deprecated field does not bring an error in the plan?

Reply:
yes we can add this case

Reply:
included in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1664